cfgov/jinja2/v1/_layouts/base.html

```diff
@@ -108,8 +108,9 @@
 -->
 
 {% block css %}
-<!--[if lt IE 9]>      <link rel="stylesheet" href="{{ static('css/main.ie.css') }}"> <![endif]-->
-<!--[if gt IE 8]><!--> <link rel="stylesheet" href="{{ static('css/main.css') }}"> <!--<![endif]-->
+<!--[if lt IE 9]><link rel="stylesheet" href="{{ static('css/main.ie8.css') }}"><![endif]-->
+<!--[if IE 9]><link rel="stylesheet" href="{{ static('css/main.ie9.css') }}"><![endif]-->
+<!--[if gt IE 9]><!--><link rel="stylesheet" href="{{ static('css/main.css') }}"><!--<![endif]-->
```

Comment:
I don't think the `<!-->` hack is necessary anymore. If you want to lose a good part of your day, read through https://github.com/h5bp/html5-boilerplate/issues/1437 (I don't recommend it, though. It's fine to leave the hack.)